CHANGELOG.md

```diff
@@ -26,46 +26,9 @@
 All notable changes to this project will be documented in this file.
 
 ## Unreleased
-* ### ALL
-    * #### Added
-    * #### Changed
-    * #### Removed
-* ### `nidcpower` (NI-DCPower)
-    * #### Added
-    * #### Changed
-    * #### Removed
-* ### `nidigital` (NI-Digital Pattern Driver)
-    * #### Added
-    * #### Changed
-    * #### Removed
-* ### `nidmm` (NI-DMM)
-    * #### Added
-    * #### Changed
-    * #### Removed
-* ### `nifgen` (NI-FGEN)
-    * #### Added
-    * #### Changed
-    * #### Removed
-* ### `nimodinst` (NI-ModInst)
-    * #### Added
-    * #### Changed
-    * #### Removed
-* ### `niscope` (NI-SCOPE)
-    * #### Added
-    * #### Changed
-    * #### Removed
-* ### `niswitch` (NI-SWITCH)
-    * #### Added
-    * #### Changed
-    * #### Removed
-* ### `nise` (NI Switch Executive)
-    * #### Added
-    * #### Changed
-    * #### Removed
 * ### `nitclk` (NI-TClk)
     * #### Added
-    * #### Changed
-    * #### Removed
+         * Added nitclk_niscope.py to demonstrate homogenous triggering.
```

Comment:
`An example, nitclk_niscope.py, to demonstrate homogeneous triggering.`

Also make sure the example name matches what ends up being actually used. (Applies to PR title too)